r/README.Rmd

```diff
@@ -161,6 +161,26 @@ remotes::install_github("romainfrancois/decor")
 install.packages(c("dplyr", "purrr", "glue"))
 ```
 
+We use Google C++ style in our C++ code. Check for style errors with
+
+```
+./lint.sh
+```
+
+Fix any style issues before committing with
+
+```
+./lint.sh --fix
+```
+
+The lint script requires Python 3 and `clang-format`. If the command isn't
+found, you can explicitly provide the path to it like
+`CLANG_FORMAT=$(which clang-format) ./lint.sh`
+
+For consistency, it is recommended that you use `clang-format-7`. On macOS,
```

Comment:
The issue is more serious than that. The behavior of clang-format changes slightly from LLVM major version to major version, so people _need_ to use clang-format-7